programmers/체육복.js

```diff
@@ -0,0 +1,16 @@
+function solution(n, lost, reserve) {
+    let realLost = lost.filter(i => reserve.indexOf(i) === -1).sort((a, b) => a - b);
+    let realReserve = reserve.filter(i => lost.indexOf(i) === -1).sort((a, b) => a - b);
+    const studentHave = n - realLost.length; 
+    const spare = [];  
+    for (let i = 0; i < realLost.length; i++) {
+        for (let j = 0; j < realReserve.length; j++) {
+            if (realReserve[j] + 1 === realLost[i] || realLost[i] === realReserve[j] - 1 || realLost[i] === realReserve[j]) {
+                spare.push(realLost[i])
+                realReserve[j] = -30
+                realLost[i] = -30   
```

Comment:
빌려준 학생들을 표시해주기 위해 -30을 해준것인데, 단순히 숫자만 보고서는 의미 파악이 힘듭니다.
-30이라는 고정값을 최상단에 선언해주고 변수명을 사용해주는 것이 좋습니다.
```js
const TOTAL_MEMBER = 30;
```
이와 같이 선언해준 뒤 아래처럼 변수명을 끌어다 쓰는 것이 훨씬 훨씬 훨씬 백만 오백만배 좋은 코드입니다.

그리고 세미콜론은 무조건 해야합니다.
그것이 자스 컨벤션입니다.
```suggestion
                realReserve[j] = -TOTAL_MEMBER;
                realLost[i] = -TOTAL_MEMBER;
```